src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -112,6 +112,7 @@ public static class EnvironmentSettingNames
         public const string LinuxAzureAppServiceStorage = "WEBSITES_ENABLE_APP_SERVICE_STORAGE";
         public const string CoreToolsEnvironment = "FUNCTIONS_CORETOOLS_ENVIRONMENT";
         public const string RunningInContainer = "DOTNET_RUNNING_IN_CONTAINER";
+        public const string AllowCorsConfiguration = "ALLOW_CORS_CONFIGURATION";
```

Comment:
Question: Should we use the "FUNCTIONS_" or "WEBSITE_" prefix to be more specific (to prevent conflicts). The current name is more generic.

Reply:
We previously got bitten by using generic env variable names. See https://github.com/Azure/azure-functions-host/issues/9504

Reply:
That is good point, let me update the variable name.

Reply:
Prefixing is a good idea (`FUNCTIONS_` would be preferrable.

We should use "Enable" here instead of "Allow". Same applies to the extension method exposed.